README.md

```diff
@@ -90,6 +90,28 @@ An iPhone 7+ and iOS 11+ are required in order to use NFC with iOS devices.
 <string>com.apple.developer.nfc.readersession.iso7816.select-identifiers</string>
 ```
 
+#### iOS Conciderations
+
+If you are having issues reading Mifare 1k Classic cards - the chances are the issue is not with this library, but with Apple's API.
+
+On iOS 11, apple released the ability to READ NFC NDEF data only using the NfcNdefReaderSession API (https://developer.apple.com/documentation/corenfc/nfcndefreadersession)
+
+A Mifare 1k Classic card will scan if there is a valid NDEF record on it. A blank card will not scan.
+
+In iOS 11, it was not possible to obtain the CSN (serial number/identity) from NFC tags/card.
+
+With iOS 13, along came the ability to write NDEF data AND read serial numbers. However, rather then adapting the NfcNdefReaderSession API, Apple created a NEW API called NfcTagReaderSession (https://developer.apple.com/documentation/corenfc/nfctagreadersession) and left the old NfcNdefReaderSession API untouched.
+
+The new NfcTagReaderSession API in iOS 13 no longer supports Mifare Classic 1k cards period. No idea why - but if you look at Apple's Dev Forums multiple people have spotted the same thing.
+
+So even if you have a Mifare Classic 1k card which reads fine with the old iOS 11 NfcNdefReaderSession API, that same card will not even scan with iOS 13's NfcTagReaderSession API.
+
+If you need to read NDEF data off of a Mifare 1k Classic card, then you need to use version 0.1.11 of this library as it was written with the NfcNdefReaderSession API. Any subsequent versions moved over to the NfcTagReaderSession API
+
+Unfortunately, even with iOS 13, it seems there is no way to read the serial number / CSN off of a Mifare Classic 1k card.
+
+Other types of NFC tags should work. 
```

Comment:
There's a lot of NFC Tags types, and probably a lot that don't work. Do you think we should probably remove this line?

Reply:
Yes!

Reply:
Done

Reply:
Looks good to me